src/utils/map-state.js

```diff
@@ -348,13 +348,13 @@ export default class MapState {
   }
 
   _unproject(pos) {
-    const viewport = new PerspectiveMercatorViewport(this._viewportProps);
-    return pos && viewport.unproject(pos, {topLeft: false});
```

Comment:
This was a bug in the older versions of viewport-mercator-project. `topLeft` should be `true`.